AntWars/ConfigurationPanel.cs

```diff
@@ -388,6 +382,31 @@ private void btn_gameConfigNew_Click(object sender, EventArgs e)
         {
             configLoader.newGame();
             configLoadedOrNewCreatedGame();
+            getStandardGameValues();
+        }
+
+        private void getStandardGameValues()
```

Comment:
``configLoader.newGame();`` dadrin sollten die standart values gesetzt werden, das erspart das du unten das immer auch in die config setzten muss, da die config ja danach geladen wird.